mimesis/providers/person.py

```diff
@@ -238,11 +241,14 @@ def password(self, length: int = 8, hashed: bool = False) -> str:
         else:
             return password
 
-    def email(self, domains: Union[tuple, list] = None) -> str:
+    def email(self, domains: Union[tuple, list] = None,
+              unique: bool = False) -> str:
         """Generate a random email.
 
         :param domains: List of custom domains for emails.
         :type domains: list or tuple
+        :param unique: Flag to generate a unique email.
+        :type unique: bool
```

Comment:
Actually, we do not need to add type to the docstrings, because we use sphinx-type-hints, so you can remove lines starts with `type:`.

Reply:
Noted. I will remove type from the docstrings.